src/TermMetaAdminUI.php

```diff
@@ -37,10 +37,10 @@ abstract class TermMetaAdminUI implements Registrable {
 	 *
 	 * @since 1.0.0
 	 *
-	 * @param \Required\Common\Taxonomy $taxonomy  The taxonomy.
-	 * @param \Required\Common\TermMeta $term_meta The term meta.
+	 * @param string $taxonomy  The taxonomy name.
+	 * @param string $term_meta The term meta key.
 	 */
-	public function __construct( Taxonomy $taxonomy, TermMeta $term_meta ) {
+	public function __construct( string $taxonomy, string $term_meta ) {
```

Comment:
I'm not supporting this change. Dependency injection is preferred as it give's us benefits like type hinting.

Reply:
I can understand that. I would prefer to separate the registration of `TermMetaAdminUI` from the Taxonomy registration. What I see is also that additional information in `TermMeta` is registered differently with `taxonomy()` instead of using the construct. I would be good to do it the same everywhere.